### ISSUE TITLE
Load arrow manipulation wheel needs to be improved (even for unrotated parts)

### ISSUE THREAD

**Issue by rickdalgarno** [tetonsim/is-cura-ui#446]
The load arrow manipulation wheel does not always behave normally, even for parts that are not rotated on the build plate. See the video in the link below. This behavior is confusing for users and, in some cases, it is not possible to orient the load in the desired direction.

The files for this issues are under NDA so are not attached to this issue. They can be accessed using this link: https://tetoncompositescom.sharepoint.com/:f:/s/TetonSimulation/EhDrY7Z0XRRIroNX2EBSDhYBhyOqoKxjj_iRuTc4Inhu2w?e=WxwPPu

Expected behavior: the load arrow follows the movement of the cursor when using the manipulator wheel. 

See #315 for a similar issue.

**Comment by dougkenik** [comment]
@rickdalgarno - can you manipulate the wheel after you change the view orientation?

**Comment by rickdalgarno** [comment]
> @rickdalgarno - can you manipulate the wheel after you change the view orientation?

No. Changing the view orientation does not change the wheel manipulation behavior.

**Comment by rickdalgarno** [comment]
@dougkenik Is this planned to be fixed for r20.2 or is it a known issue?

**Comment by dougkenik** [comment]
It's going to take some effort to fix. I believe (from what I can see) it is something in the QML code base which does not update the mouse location correctly. I would say let's push

**Comment by rickdalgarno** [comment]
That is fine with me. It is a common issue, though, and would be good to prioritize for r20.3.

**Comment by rickdalgarno** [comment]
Just another friendly reminder that the load wheels can be very difficult to adjust and behave erratically. This behavior is fairly common in my experience (aka not hard to trigger).

**Comment by rickdalgarno** [comment]
@dougkenik I had a support call with Jason from RE Suspension today and he told me that the load angle manipulation wheel was very hard to use in some cases. Another data point for getting this fixed in our next release.
